package.json

```diff
@@ -38,9 +38,7 @@
     "url": "https://github.com/netlify/cli/issues"
   },
   "scripts": {
-    "build": "run-s build:*",
-    "build:site": "cd site && npm install && npm run build",
-    "ci": "run-s test build",
+    "build:site": "cd site && npm ci && npm run build",
```

Comment:
Would `npm install` be better than `npm ci` when running this locally?

Reply:
I would actually always opt for `npm ci` (so developers always have the same dependencies installed), unless one is explicitly trying to update dependencies.
`npm install` will update lock file if it doesn't match with `package.json`. I would prefer to fail in that case.

Reply:
This makes sense. :+1: